src/java/org/jivesoftware/sparkimpl/plugin/systray/SysTrayPlugin.java

```diff
@@ -58,6 +59,7 @@
 import org.jivesoftware.sparkimpl.settings.local.SettingsManager;
 import org.jivesoftware.smack.chat.ChatManagerListener;
 import org.jivesoftware.smackx.chatstates.ChatStateListener;
+import org.lobobrowser.html.domimpl.HTMLElementBuilder;
```

Comment:
Yeah, that import should not be there . Im not sure how that got in there ! Ill remove it!